id-repository/credential-request-generator/src/main/java/io/mosip/credential/request/generator/constants/ApiName.java

```diff
@@ -7,6 +7,7 @@
  */
 public enum ApiName {
 
-	CRDENTIALSERVICE;
+	CRDENTIALSERVICE,
+	KERNELAUDITMANAGER;

```

Comment:
this also need the config to be added in the property files ?

Reply:
It is added in property file in mosip-config branch qa3-1.1.5